### DISCUSSION TITLE
Upgrades of third-party services

### DISCUSSION THREAD

**Discussion by dominikriemer** [apache/streampipes#1215] [Dev]
Hi everyone,

just a quick note that I have updated our third-party services Consul, CouchDB and InfluxDB to the most recent version. 

There are some environment variables you might want to change in your dev setup, see the variables defined in the `DefaultEnvironment` of `streampipes-commons`.
In addition, in case you have a dev system up and running and want to migrate data stored in Influx from v1 to v2, change the config in the compose/cli/k8s file: ` DOCKER_INFLUXDB_INIT_MODE=${SP_INFLUX_INIT_MODE}` should be either `setup` or `upgrade`.

Please let me know in case you have any issues as there still might be things that are working as expected, this was quite a heavy upgrade ;-)

**Comment by flomickl** [comment]
Oh, sorry, I did not see this post and created the following thread
but here would be a better place for it:

https://github.com/apache/streampipes/issues/1221#issue-1570039264
